tests/general/Makefile.am

```diff
@@ -67,7 +67,7 @@ TESTS = run_tests.sh
 endif # RUN_TESTS
 
 %.F90: %.F90.in
-	util/pio_tf_f90gen.pl --annotate-source --out=$@ $<
+        util/pio_tf_f90gen.pl --annotate-source --out=$@ $<
```

Comment:
This needs to be a tab char, not spaces.

Reply:
thanks